SC-081: Introduce Schedule of Reducing Validity and Data Reuse Periods (#553)
cabforum/documents: docs/BR.md

Applicant information MUST include, but not be limited to, at least one Fully-Qualified Domain Name or IP address to be included in the Certificate's `subjectAltName` extension.

[Section 6.3.2](#632-certificate-operational-periods-and-key-pair-usage-periods) limits the validity period of Subscriber Certificates.

The CA MAY use the documents and data provided in [Section 3.2](#32-initial-identity-validation) to verify certificate information, or may reuse previous validations themselves, provided that the CA obtained the data or document from a source specified under [Section 3.2](#32-initial-identity-validation) or completed the validation itself within the maximum number of days prior to issuing the Certificate, as defined in the following table:

> Table: Maximum Allowed Non-SAN Validation Data Reuse Period

Reviewer: Is this table the correct one for validations relating to `dNSName`s and `iPAddress`es included in the `permittedSubtrees` field of the `nameConstraints` extension within a technically constrained CA certificate? Intuitively, it should not, but the wording of this section indicates that it is.

| __Certificate issued on or after__ | __Certificate issued before__ | __Maximum number of days for data reuse__ |
| --                                 | -                             | -                                         |

> |                                    | September 15, 2025            | 825                                       |
> | September 15, 2025                 | September 15, 2026            | 366                                       |

Reviewer 2: suggestion: either remove the "certificate issued before" date from the last row, or add more rows which further reduce data reuse periods on dates in lock-step with the other tables below. Otherwise it's unclear what happens after Sept 15 2026.
```suggestion
|                                    | September 15, 2025            | 825                                       |
| September 15, 2025                 |                               | 366                                       |
```
```suggestion
|                                    | September 15, 2025            | 825                                       |
| September 15, 2025                 | September 15, 2026            | 366                                       |
| September 15, 2026                 | September 15, 2027            | 200                                       |
| September 15, 2027                 |                               | 100                                       |
```

Reviewer 3: I interpreted it to mean "no reuse of non-SAN validation after September 15, 2026," which I am hoping is not the case.

Reviewer 4: > I interpreted it to mean "no reuse of non-SAN validation after September 15, 2026," which I am hoping is not the case.

@clintwilson could you clarify this point please? What is supposed to happen after 15.09.2026?

> For validation of Domain Names and IP Addresses according to [Section 3.2.2.4](#3224-validation-of-domain-authorization-or-control) and [Section 3.2.2.5](#3225-authentication-for-an-ip-address), any data, document, or completed validation used MUST be obtained within the maximum number of days days prior to issuing the Certificate, as defined in the following table:

Reviewer 5: s/days days/days/
```suggestion
For validation of Domain Names and IP Addresses according to [Section 3.2.2.4](#3224-validation-of-domain-authorization-or-control) and [Section 3.2.2.5](#3225-authentication-for-an-ip-address), any data, document, or completed validation used MUST be obtained within the maximum number of days prior to issuing the Certificate, as defined in the following table:
```

Table: Maximum Allowed SAN Validation Data Reuse Period

| __Certificate issued on or after__ | __Certificate issued before__ | __Maximum number of days for data reuse__ |
| --                                 | -                             | -                                         |

> |                                    | September 15, 2025            | 398                                       |
> | September 15, 2025                 | September 15, 2026            | 200                                       |
> | September 15, 2026                 | April 15, 2027                | 100                                       |
> | April 15, 2027                     | September 15, 2027            | 45                                        |
> | September 15, 2027                 |                               | 10                                        |

Reviewer 2: suggestion: stick to just 3 effective dates, on Sept 15 of the next three years. I think the inclusion of the April date makes it harder to think about this ballot's timeline, without providing a commensurate benefit.
```suggestion
|                                    | September 15, 2025            | 398                                       |
| September 15, 2025                 | September 15, 2026            | 200                                       |
| September 15, 2026                 | September 15, 2027            | 100                                       |
| September 15, 2027                 |                               | 10                                        |
```

In no case may a prior validation be reused if any data or document used in the prior validation was obtained more than the maximum time permitted for reuse of the data or document prior to issuing the Certificate.

> After the change to any validation method specified in the Baseline Requirements or EV Guidelines, a CA may continue to reuse validation data or documents collected prior to the change, or the validation itself, for the period stated in this [Section 4.2.1](#421-performing-identification-and-authentication-functions) unless otherwise specifically provided in a ballot.

Reviewer 5: ```suggestion
After the change to any validation method specified in the Baseline Requirements or EV Guidelines, a CA may continue to reuse validation data or documents collected prior to the change, or the validation itself, for the period stated in [Section 4.2.1](#421-performing-identification-and-authentication-functions) unless otherwise specifically provided in a ballot.
```

### 6.3.2 Certificate operational periods and key pair usage periods

Subscriber Certificates issued on or after 1 September 2020 and before 15 September 2025 SHOULD NOT have a Validity Period greater than 397 days and MUST NOT have a Validity Period greater than 398 days.

Subscriber Certificates issued on or after 15 September 2025 and before 15 September 2026 SHOULD NOT have a Validity Period greater than 199 days and MUST NOT have a Validity Period greater than 200 days.

Subscriber Certificates issued on or after 15 September 2026 and before 15 April 2027 SHOULD NOT have a Validity Period greater than 99 days and MUST NOT have a Validity Period greater than 100 days.

Subscriber Certificates issued on or after 15 April 2027 SHOULD NOT have a Validity Period greater than 44 days and MUST NOT have a Validity Period greater than 45 days.

> Table: Reference for maximum Validity Periods of Subscriber Certificates

Reviewer 5: Nit: The capitalization is not consistent with the table labels above.

> | __Certificate issued on or after__ | __Certificate issued before__ | __Maximum number of days for data reuse__ |

Reviewer 6: @clintwilson  I presume you mean the below:

```suggestion
| __Certificate issued on or after__ | __Certificate issued before__ | __Maximum number of days for validity__ |
```

Author: Yup! I even have this fixed locally, just forgot to push the commit 🤦‍♀️

| --                                 | -                             | -                                         |

> | September 1, 2020                  | September 15, 2025            | 398                                       |
> | September 15, 2025                 | September 15, 2026            | 200                                       |
> | September 15, 2026                 | April 15, 2027                | 100                                       |
> | April 15, 2027                     |                               | 45                                        |

Reviewer 2: suggestion: stick to the three Sept 15 dates used above, again to make this ballot very easy to think about. Also, remove the "September 1, 2020" note in the first row, as no Subscriber Certificates issued before that date are still valid.
```suggestion
|                                    | September 15, 2025            | 398                                       |
| September 15, 2025                 | September 15, 2026            | 200                                       |
| September 15, 2026                 | September 15, 2027            | 100                                       |
| September 15, 2027                 |                               | 45                                        |
```

Subscriber Certificates issued on or after 15 March 2026 and before 15 March 2027 SHOULD NOT have a Validity Period greater than 199 days and MUST NOT have a Validity Period greater than 200 days.

Subscriber Certificates issued on or after 15 March 2027 and before 15 March 2029 SHOULD NOT have a Validity Period greater than 99 days and MUST NOT have a Validity Period greater than 100 days.

> Subscriber Certificates issued on or after 15 March 2029 SHOULD NOT have a Validity Period greater than 46 days and MUST NOT have a Validity Period greater than 47 days.

Reviewer 7: Did you also mean to change the years for validation data reuse (lines 1211-1213)?

Author: I didn't at the time, but following additional discussion in the Servercert Working Group, have aligned those dates with this change.

Subscriber Certificates issued on or after 15 March 2026 and before 15 March 2027 SHOULD NOT have a Validity Period greater than 199 days and MUST NOT have a Validity Period greater than 200 days.

Subscriber Certificates issued on or after 15 March 2027 and before 15 March 2029 SHOULD NOT have a Validity Period greater than 99 days and MUST NOT have a Validity Period greater than 100 days.

Subscriber Certificates issued on or after 15 March 2029 SHOULD NOT have a Validity Period greater than 46 days and MUST NOT have a Validity Period greater than 47 days.

Table: Reference for maximum Validity Periods of Subscriber Certificates

| __Certificate issued on or after__ | __Certificate issued before__  | __Maximum Validity Period__  |
| --                                 | --                             | --                           |
|                                    | March 15, 2026                 | 398 days                     |
| March 15, 2026                     | March 15, 2027                 | 200 days                     |
| March 15, 2027                     | March 15, 2029                 | 100 days                     |

> | March 15, 2029                     |                                | 47 days                      |

Reviewer 8: dm!!! apple.
